docker/add-user.sh

```diff
@@ -7,15 +7,14 @@
 
 set -e
 
-read -r -p "Email (full user@domain format): " EMAIL
+test -n "${EMAIL:-}" || read -r -p "Email (full user@domain format): " EMAIL
```

Comment:
Can you replace this with:

```
if test -z "${EMAIL}"; then
        read -r -p "Email (full user@domain format): " EMAIL
fi
```

It should be the same effect, and I am okay with the change, I just find this form more readable in this case.

Is that okay with you? What do you think?

Thank you!

Reply:
Done